examples/typescript/provisioner/__tests__/main-test.ts

```diff
@@ -0,0 +1,85 @@
+// import { Testing } from "cdktf";
+// import "cdktf/lib/testing/adapters/jest";
+
+describe("My CDKTF Application", () => {
+  it.todo("should be tested");
```

Comment:
nit: we could also remove this file, but no strong opinion here

Reply:
We have them in all examples since they should be similar to the template